src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.cs

```diff
@@ -280,6 +284,41 @@ private static Glyph GetGlyph(SymbolKind kind, Accessibility? declaredAccessibil
             return result.ToImmutable();
         }
 
+        private static void AddNamesFromExistingOverloads(CSharpSyntaxContext context, ArrayBuilder<(string name, SymbolKind kind)> result, CancellationToken cancellationToken)
+        {
+            var parameterSyntax = context.LeftToken.GetAncestor(n => n.IsKind(SyntaxKind.Parameter)) as ParameterSyntax;
+            if (parameterSyntax is not { Type: { } parameterType, Parent: { Parent: MethodDeclarationSyntax method } })
+            {
+                return;
+            }
+
+            if (method.Parent is null)
+                return;
+
+            var methodParameterType = context.SemanticModel.GetTypeInfo(parameterType, cancellationToken).Type;
+            if (methodParameterType is null)
+                return;
+
+            var siblingMethods = method.Parent.ChildNodes().OfType<MethodDeclarationSyntax>().Where(m => m != method && m.Identifier.Text == method.Identifier.Text);
```

Comment:
Should this be using `ValueText`?

```suggestion
            var siblingMethods = method.Parent.ChildNodes().OfType<MethodDeclarationSyntax>().Where(m => m != method && m.Identifier.ValueText == method.Identifier.ValueText);
```